src/message.rs

```diff
@@ -79,9 +79,18 @@ impl RunnerState {
                 }
             }
             RunnerState::Exited(err) => {
+                // If the err type was an UnexpectedExit then we would already have stdout and
+                // stderr from the error message so default to that instead
+                // The Error enum doesn't support .clone() so have to do some weird object
```

Comment:
Not sure if you can add `#[derive(Clone)]` annotation to the current Error enum. I think it might of the random errors it wraps might be uncloneable, making it something that we can't `#[derive(Clone)]` for - (part of why I wanted to see the entire error type replaced with something simpler... anyhow, no concrete suggestion here, more of the thing that might drive me crazy enough to tinker with a refactor on top of your PR. :-D